storage_service/locations/models/space.py

```diff
@@ -216,6 +216,22 @@ def post_move_to_storage_service(self, *args, **kwargs):
             # This is optional for the child class to implement
             pass
 
+    def _move_from_path_mangling(self, staging_path, destination_path):
```

Comment:
I know this is a private function, but mind adding a docstring that provides examples of what the mangling looks like?